src/main/java/uk/gov/hmcts/darts/utilities/deserializer/LocalDateTimeTypeDeserializer.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.darts.utilities.deserializer;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class LocalDateTimeTypeDeserializer extends JsonDeserializer<LocalDateTime> {
+
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("d:MMM:uuuu HH:mm:ss");
+
+    @Override
+    public LocalDateTime deserialize(JsonParser parser, DeserializationContext ctxt) throws IOException, JacksonException {
```

Comment:
In general, I think the team needs to question any use of `LocalDateTime` in the code as the server system clock will be in UTC.
We deal with UTC timestamps in darts-api and currently you can expect to always get a UTC date stored in the database and returned in the darts-api response.
If we wish get a "local" UK time, then I think it would be less confusing for us to use `ZonedDateTime` like we have done in some places where we needed the local time in darts-api for some metadata files as part of the audio download.  See `DateConverters` ZonedDateTime offsetDateTimeToLegacyDateTime(final OffsetDateTime offsetDateTime).

In darts-gateway it would be worth someone looking at the other method `offsetDateTimeFrom` as it uses LocalDateTime and I suspect it isn't what we want?

Reply:
I totally agree. Can we look at that in another ticket? My aim here was to replicate what we had with gson